distributor-api-v1/operations.md

```diff
@@ -151,20 +151,107 @@ Preferred initial call used to obtain all static data about distributor configur
 
 |  | Property | Type | Description |
 | :--- | :--- | :--- | :--- |
-| `Cities` | array of [City](operations.md#city) | required | Cities supported by hotel. |
+| `NowUtc` | string | required | UTC Time when configuration was asked for. |
+| `LanguageCode` | string | required | Language code used for Distributor. |
+| `CurrencyCode` | string | required | Currency code used for Distributor. |
+| `IntroVideoUrl` | string | required | Url to intro video for Distributor. |
+| `Theme` | string | required | Distributor's [Theme](operations.md#theme). |
+| `PrimaryColor` | string | required | Distributor's primary color in Hex format.  |
+| `VoucherCode` | string | required | Voucher code enabling special rate offerings. |
+| `DisplayVoucherCode` | boolean | optional | Information whether enterprise's voucher codes should be listed in Distributor (voucher codes are listed by default). |
+| `GtmContainerId` | string | required | Google tag manager indentifier. |
```

Comment:
IMHO it can be optional. GTM is not mandatory, or is it?

Reply:
@vitasamek I'd like to try to finish this, so if you can be a little patient with, if I ask stupid questions 😄 .

Does it mean it's optional when it can be empty string, but not null/undefined? I'd say no?

And for the other fields, can we just rely on types described here:
```
src/Mews.Server.Web/Api/Distributor/V1/Dto/Configuration/Configuration.cs
```
?

Reply:
For now i've written in a way even empty string means optional and you were right gtm id doesn't need to be filled in.